sous-chef/tests/integration_tests/messaging/test_gsheets_api.py

```diff
@@ -17,7 +17,7 @@ class TestGSheetsHelper:
     def test_write_and_read_worksheet(gsheets_helper):
         workbook_name = random_string(12)
         worksheet_name = "test"
-        fake_df = pd.DataFrame({"Vars": ["a", "b"], "Vals": [1.1, 2.4]})
+        fake_df = pd.DataFrame({"Vars": ["a", "b"], "Vals": ["1.1", "2.4"]})
```

Comment:
@AkagiShigeru  this is required if we always have numerize as false, but should be updated if we fix it as i proposed

Reply:
yes, agree